adodb-loadbalancer.inc.php

```diff
@@ -174,7 +174,7 @@ public function removeConnection($i)
      * @param  string $type Type of database connection, either: 'write' capable or 'readonly'
      * @return bool|int|string
      */
-    private function getConnectionByWeight($type)
+    public function getConnectionByWeight($type)
```

Comment:
This is the same situation as _getConnection() above.

Reply:
Again, no objections with changing the method's visibility. 

I'm only suggesting to introduce class constants to avoid the use of hardcoded strings for $type. This is actually not directly related to this work item, more of a general improvement / best practice.

Reply:
Normally I would agree, but there are multiple classes involved ( ADOdbLoadBalancerConnection and ADOdbLoadBalancer ) and using a class constant would complicate things unnecessarily, since those classes do not extend from one another. A global constant (prefixed of course) would be the simplest in this case, but is it really worth it?

Reply:
Right, I did not consider that there were multiple classes involved. Let's leave it at that.